src/content/en/updates/2016/11/nic55.md

```diff
@@ -0,0 +1,177 @@
+          } else {
+            console.error('xhr failed', xhr.statusText);
+          }
+        }
+      };
+      xhr.onerror = function (e) {
+        console.error(xhr.statusText);
+      };
+      xhr.open('GET', url);
+      xhr.send();
+    }
+
+Re-writing it with `promises` helps avoid the nesting problem:
+
+    function logFetch(url) {
+      return fetch(url)
+        .then(response => response.text())
+        .then(text => {
+          console.log(text);
+        }).catch(err => {
+          console.error('fetch failed', err);
+        });
+    }
+
+But, Promise-based code can still be difficult to read when there are long
+chains of asynchronous dependencies. 
+
+Chrome now supports the `async` and `await` JavaScript keywords, allowing you
+to write Promise-based JavaScript that can be as structured and
+readable as synchronous code.
+
+Instead, our asynchronous function can be simplified to this:
+
+    async function logFetch(url) {
+      try {
+        const response = await fetch(url);
+        console.log(await response.text());
+      }
+      catch (err) {
+        console.log('fetch failed', err);
+      }
+    }
+
+Jake has a great post: 
+[Async Functions - making promises friendly](/web/fundamentals/getting-started/primers/async-functions)
+with all the details.
+
+## Persistent Storage {: #persistent-storage }
+
+The persistent storage origin trial is now over. You can now mark web
+storage as persistent, preventing Chrome from automatically clearing the
+storage for your site.
+
+    if (navigator.storage && navigator.storage.persist) {
+      navigator.storage.persist().then(granted => {
+        if (granted)
+          alert("Storage will not be cleared except by explicit user action");
+        else
+          alert("Storage may be cleared by the UA under storage pressure.");
+      });
+    }
+
+In addition, sites that have high engagement, have been added to the
+home screen or have push notifications enabled are automatically
+granted the persistence permission.
+
+Check out Chris Wilson’s [Persistent Storage](/web/updates/2016/06/persistent-storage)
+post for full details and how you can request persistent storage for your site.
+
+
+## CSS Automatic Hyphenation
+
+[CSS automatic hyphenation](/web/updates/2016/10/css-hyphens), one of Chrome’s
+most frequently requested [layout features](https://googlechrome.github.io/samples/css-hyphens/)
+is now supported on Android and Mac. 
+
+## Web Share API
+
+And finally, it’s now easier to invoke native sharing capabilities with the
+new [Web Share API](https://github.com/mgiuca/web-share/blob/master/docs/interface.md),
+which is available as an
+[origin trial](https://github.com/jpchase/OriginTrials/blob/gh-pages/developer-guide.md).
+Paul (Mr. Web Intents) Kinlan has all the details in his
+[Navigator Share](/web/updates/2016/10/navigator-share) post.
+
+## Closing
+
+These are just a few of the changes in Chrome 55 for developers.  
+
```

Comment:
I should have caught this in the script draft. Nit, should be: "These are just a few of the changes for developers in Chrome 55."